pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -208,20 +208,36 @@ type PermittedHostDevices struct {
 // PciHostDevice represents a host PCI device allowed for passthrough
 // +k8s:openapi-gen=true
 type PciHostDevice struct {
-	PCIVendorSelector        string `json:"pciVendorSelector"`
-	ResourceName             string `json:"resourceName"`
-	ExternalResourceProvider bool   `json:"externalResourceProvider,omitempty"`
+	// a combination of a vendor_id:product_id required to identify a PCI device on a host.
+	PCIVendorSelector string `json:"pciVendorSelector"`
```

Comment:
This is probably not for us, but PCIVendorSelector here looks a bit misleading.
In `pciVendorSelector: "10DE:1DB6"` `10DE` is the Vendor ID (`10DE` = NVIDIA Corporation) while `1DB6` is the Device ID inside the vendor catalog (`1DB6` is Tesla V100 PCIe 32GB for NVIDIA Corporation) . 
And we are going to select the single device not all the devices from a single vendor so this looks misleading to me.

Reply:
@tiraboschi - so what do we want to do with it? do you want to change the naming in HCO to be more accurate, or to keep it as is, to match the KubeVirt API?

Reply:
@ashleyschuett, @enp0s3 - could you please take a look?

Reply:
@nunnatsa @tiraboschi Indeed the name is misleading, but on the other hand if we change the name now in Kubevirt it will trigger API change and we will have to implement backward compatibility policy (currently we don't have such a webhook).

Reply:
we introduced this API on HCO side only now and we are still to release it so I think it will be better to fix it here and now (even if will be named differently than on kubevirt API) to avoid having to handle backward compatibility in a future release.

Reply:
@tiraboschi +1